NuSpec/Build/RedGate.AppHost.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup Condition="$(RedGateAppHostOutputPath) == ''">
+        <RedGateAppHostOutputPath>$(TargetDir)</RedGateAppHostOutputPath>
+    </PropertyGroup>
+    <Target Name="CopyRedGateAppHostFiles" AfterTargets="Build">
+        <ItemGroup>
+            <RedGateAppHostFiles Include="$(MSBuildThisFileDirectory)\..\tools\**\*.*" />
+        </ItemGroup>
+        <Copy SourceFiles="@(RedGateAppHostFiles)" DestinationFiles="@(RedGateAppHostFiles->'$(RedGateAppHostOutputPath)\%(RecursiveDir)%(Filename)%(Extension)')" SkipUnchangedFiles="true" />
```

Comment:
And from http://stackoverflow.com/a/30316946 and associated answers I'd go for the `CopyToOutputDirectory` method instead. Be right back